### PR TITLE
Adding tools, weapons, and consumables

### PR DESCRIPTION
**Pull request by Godimas**
Yoloing this all in at the same time. Because I CLEARLY know what im doing. Please dont ask me if I know what im doing. Anyways, all three categories are in, and I renames items to Misc. items

### CODE REVIEW

File: Other Goodies/Mods/LCDInfo [hunk 1]
```diff
@@ -542,10 +578,61 @@ namespace EconomySurvival.LCDInfo
...
+                MyDefinitionId.TryParse(item.Value.item.Type.TypeId, item.Value.item.Type.SubtypeId, out myDefinitionId);
+
+                WriteTextSprite(ref frame, myDefinitions[myDefinitionId].DisplayNameText, position, TextAlignment.LEFT);
+                WriteTextSprite(ref frame, KiloFormat(item.Value.amount), position + right, TextAlignment.RIGHT);
+
+                position += newLine;
+            }
+        }
+		
+        void DrawAmmoSprite(ref MySpriteDrawFrame frame, ref Vector2 position, IMyTextSurface surface)
```
Comment (Beersatron) [inline Other Goodies/Mods/LCDInfo:599]: function name is repeated!

File: Other Goodies/Mods/LCDInfo [hunk 2]
```diff
@@ -542,10 +578,61 @@ namespace EconomySurvival.LCDInfo
...
```
Comment (Beersatron) [inline Other Goodies/Mods/LCDInfo:616]: function name is repeated!!

File: Other Goodies/Mods/LCDInfo [hunk 3]
```diff
@@ -192,6 +198,27 @@ namespace EconomySurvival.LCDInfo
 
                         cargoAmmos[name].amount += amount;
                     }
+                    else if (type == "Tools")
+                    {
+                        if (!toolsAmmos.ContainsKey(name))
```
Comment (Beersatron) [inline Other Goodies/Mods/LCDInfo:203]: toolsAmmos should be cargoTools ?

File: Other Goodies/Mods/LCDInfo [hunk 4]
```diff
@@ -542,10 +578,61 @@ namespace EconomySurvival.LCDInfo
...
             }
         }
+		
+        void DrawToolSprite(ref MySpriteDrawFrame frame, ref Vector2 position, IMyTextSurface surface)
+        {
+            WriteTextSprite(ref frame, "Tools", position, TextAlignment.LEFT);
+
+            position += newLine;
+
+            foreach (var item in cargoToolss)
```
Comment (Beersatron) [inline Other Goodies/Mods/LCDInfo:588]: too many s ....

File: Other Goodies/Mods/LCDInfo (hunk 1 above)
Comment (Godimas) [reply Other Goodies/Mods/LCDInfo]: Foxed

File: Other Goodies/Mods/LCDInfo (hunk 2 above)
Comment (Godimas) [reply Other Goodies/Mods/LCDInfo]: Fuxed

File: Other Goodies/Mods/LCDInfo (hunk 3 above)
Comment (Godimas) [reply Other Goodies/Mods/LCDInfo]: Faxed

File: Other Goodies/Mods/LCDInfo (hunk 4 above)
Comment (Godimas) [reply Other Goodies/Mods/LCDInfo]: Flilexed

**Comment by Godimas** [comment]
Got a +1 from the skini!

**Pull request merged**